get_databricks_cluster_eventlog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Default directory to save results to
+results_dir="$PWD/databricks_cluster_eventlogs"
+mkdir -p $results_dir
```

Comment:
Someone could invoke this script from a directory that does not contain this script. It might make for less surprises to the user if the default result dir is always alongside this script. I control for that here and actually check that directory in to source control containing only a .gitignore file to keep the contents out of source control: https://github.com/synccomputingcode/spark_log_parser/blob/main/parse.sh

If this way is preferred README should be updated to reflect it.

Reply:
Ah, I slipped on thinking $PWD was the script directory. I actually tried your linked method first, but then an error was popping up regarding the databricks cli command. I'll give it another try though. Ideally the script could be run from anywhere.

Reply:
I think you could also get away with,
```
HOME="$(dirname "$0")"
```
So long as $HOME doesn't need to be absolute

Reply:
nm, looks like copying it from GitHub keeps the tab characters in your pasteboard

Reply:
Yep this worked! The issue I was running into was reassigning HOME, so I just used a different variable name.